src/components/categories-box/CategoriesBox.jsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+
+import "./CategoriesBox.css";
+
+const CategoriesBox = () => {
+  return (
+    <div class="categories-box">
+      <div class="category">
+        <p>Games</p>
+      </div>
+      <div class="category">
+        <p>Family</p>
+      </div>
+      <div class="category">
+        <p>Nature</p>
+      </div>
+      <div class="category">
+        <p>Comedy</p>
+      </div>
+      <div class="category">
+        <p>Songs</p>
+      </div>
+      <div class="category">
+        <p>Food</p>
+      </div>
+      <div class="category">
+        <p>Motivational</p>
+      </div>
```

Comment:
Instead of repeating the same thing you can just create a component and an array of all values like ['songs','comedy',....] and use map function to pass those values as props to the created component

Reply:
change class to className

Reply:
Nice suggestion
Ok will make changes in functionality-react branch